eng/swagger_to_sdk_config_track.json

```diff
@@ -1,17 +1,13 @@
 {
   "$schema": "https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
-  "advancedOptions": {
-    "createSdkPullRequests": true,
-    "generationCallMode": "one-for-all-configs"
-  },
   "generateOptions": {
     "generateScript": {
-      "path": "pwsh ./eng/scripts/Automation-Sdk-Generate.ps1"
+      "path": "pwsh ./scripts/Automation-Sdk-Generate.ps1"
```

Comment:
It is probably worth testing to ensure this path is supposed to be relative to the json file vs the root of the repo?

Reply:
Thanks for noticing. It should be the root of the repo.

Path updated!